src/OSPSuite.Core/Serialization/Xml/StartValueXmlSerializer.cs

```diff
@@ -4,14 +4,14 @@
 
 namespace OSPSuite.Core.Serialization.Xml
 {
-   public abstract class StartValueXmlSerializer<T> : EntityXmlSerializer<T> where T : StartValueBase
+   public abstract class StartValueXmlSerializer<T> : EntityXmlSerializer<T> where T : PathAndValueEntity
```

Comment:
Rename the serializer and file please to match the base class

Reply:
It turns out that I can remove this level of abstraction all together

Can you please check it again since this change is not simply a rename of the file and class.